0001-consolidate-terraform.md

```diff
@@ -0,0 +1,52 @@
+# Consolidate Terraform
+
+* Status: proposed
+* Deciders: @ZavenArra, Peter Willis, Tuan-Ahn Nguyen, MattK
+* Date: 2022-03-01 
+
+Technical Story: Where should our terraform code be located? 
+
+## Context and Problem Statement
+
+Current terraform is spread through the treetracker-infrastructure repository.  We want to consolidate this IaC in a way that makes it easy to to find and easy for new volunteers to contribute to.  We don't want other kinds of infra codes, such as scripts or deployment jobs, to create extra mental overhead. 
+
+
+## Decision Drivers <!-- optional -->
+
+* Clean organizational strategy
+* Ease in which new volunteers (some rather junior) can find terraform codes
+* Ease in which new volunteers can contribute or create new modules
+
+## Considered Options
+
+1. Create a repository just for terraform and related utility scripts
+2. Organize treetracker-infrastructure in subfolders: terraform, application platform, utilities
+
+## Decision Outcome
+
+No choice yet
+
+Chosen option: "[option 1]", because [justification. e.g., only option, which meets k.o. criterion decision driver | which resolves force force | … | comes out best (see below)].
+
+### Positive Consequences <!-- optional -->
+
+* [e.g., improvement of quality attribute satisfaction, follow-up decisions required, …]
+* …
+
+### Negative Consequences <!-- optional -->
+
+* [e.g., compromising quality attribute, follow-up decisions required, …]
+* …
+
+## Pros and Cons of the Options <!-- optional -->
+
+### [option 1]
+
+* Good, because it keeps with one repository - one purpose
+* Bad, because dependencies with application platform servic deployments are cross repository
```

Comment:
I don't think this is necessarily bad. I prefer separation of concerns and each repo should have one focus or goal. It allows for independent development and does not tightly couple modules/projects.